### PR TITLE
Docs/beta

### PR DESCRIPTION
**Pull request by jorgeucano**
## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/scullyio/scully/blob/master/CONTRIBUTING.md#commit
- [x] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [x] Other... Please describe:

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

**Comment by jorgeucano** [comment]
@geromegrignon This is the pr

**Comment by geromegrignon** [comment]
I was going to make a PR about the docs.
If you mind updating this PR : 

The *showcase*, *utils*, and *scully* doc files don't have a *title* metadata.
It would be great to add it with a capitalized name to be consistent with the others files : 
image

Doing a quick review right now too.

**Comment by jorgeucano** [comment]
> I was going to make a PR about the docs.
> If you don't mind updating this PR :
> 
> The _showcase_, _utils_, and _scully_ doc files don't have a _title_ metadata.
> It would be great to add it with a capitalized name to be consistent with the others files :
> image
> 
> Doing a quick review right now too.

done :D

File: docs/getting-started.md [hunk 1]
```diff
@@ -9,11 +9,13 @@ Welcome to Scully!
...
 Before getting started, please read the [Prerequisites](pre-requisites.md).
 
-This getting started guide covers two topics:
+This getting started guide covers three topics:
 
 1. [Installation](#installation)
 2. [Building](#build)
 
+**_IMPORTANT:_ Scully requires the router to be present in your application. To have this automatically generated, choose the option to add Angular Routing from the prompt when running the commands below.**
+
```
Comment (geromegrignon) [inline docs/getting-started.md:18]: It was changed meanwhile by #450 as stated by @facurodriguez

File: docs/features.md [hunk 2]
```diff
@@ -3,4 +3,30 @@ title: Features
...
+- Scully Content Component
+- Transfer State Service
+- Utility Methods
+
+[Plugins System](/docs/plugins):
+
+- Router Plugins
+- Render Plugins
+- File Handler Plugins
+
```
Comment (SanderElias) [inline docs/features.md:25]: missing `alldone` and `routeDiscoveryDone` plugins.
Both are called with the complete routelist, and their result is ignored.

**Pull request merged**